src/components/AddPaymentMethodMenu.tsx

```diff
@@ -1,78 +1,74 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
+import type {RefObject} from 'react';
 import React from 'react';
+import type {View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
 import {withOnyx} from 'react-native-onyx';
-import _ from 'underscore';
+import type {ValueOf} from 'type-fest';
 import useLocalize from '@hooks/useLocalize';
 import * as ReportActionsUtils from '@libs/ReportActionsUtils';
 import * as ReportUtils from '@libs/ReportUtils';
-import iouReportPropTypes from '@pages/iouReportPropTypes';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type {AnchorPosition} from '@src/styles';
+import type {Report, Session} from '@src/types/onyx';
+import type AnchorAlignment from '@src/types/utils/AnchorAlignment';
 import * as Expensicons from './Icon/Expensicons';
 import PopoverMenu from './PopoverMenu';
-import refPropTypes from './refPropTypes';
 
-const propTypes = {
+type AddPaymentMethodMenuOnyxProps = {
+    /** Session info for the currently logged-in user. */
+    session: OnyxEntry<Session>;
+};
+
+type AddPaymentMethodMenuProps = AddPaymentMethodMenuOnyxProps & {
     /** Should the component be visible? */
-    isVisible: PropTypes.bool.isRequired,
+    isVisible: boolean;
 
     /** Callback to execute when the component closes. */
-    onClose: PropTypes.func.isRequired,
+    onClose: () => void;
 
     /** Callback to execute when the payment method is selected. */
-    onItemSelected: PropTypes.func.isRequired,
+    onItemSelected: (paymentMethod: ValueOf<typeof CONST.PAYMENT_METHODS>) => void;
```

Comment:
You can reuse existing `PaymentMethod` type
```suggestion
    onItemSelected: (paymentMethod: PaymentMethod) => void;
```